ui/imports/Themes/LightTheme.qml

```diff
@@ -25,6 +25,8 @@ Theme {
     property color darkBlue: "#3c55c9"
     property color darkBlueBtn: "#5a70dd"
     property color red: "#FF2D55"
+    property color tenPercentRed: "#19FF2D55"
+    property color twentyPercentRed: "#32FF2D55"
```

Comment:
I wouldn't bother with adding new colors to `ui/imports/Themes` as those are legacy themes.

`StatusQ.Core.Theme` is where the party's at. And we already have `dangerColor` variants for those.

Reply:
good point @PascalPrecht. Is there any plan to remove this legacy code?

Reply:
There's no "plan" as in, there's no particular roadmap we've decided on, on when to remove this. But generally, it can be removed at any time as long as there's enough spare cycles for that.

Eventually we want to get rid off all the legacy component/theme/style code. But we are, and probably will be for longer, in a transitioning phase. 

We could also remove everything in one go but experience shows that this big bang kind of efforts never end well. So I'd rather have use do it incrementally.

Reply:
Generally, you never need to ask for permission to create issues :)
Please, just go ahead with that. If we feel that some of them can be closed, it's just a click of a button